sql/schema/v1-core.sql

```diff
@@ -1684,14 +1693,38 @@ BEGIN
         'SELECT ARRAY(
             SELECT DISTINCT e.tenant_id
             FROM %I e
-            WHERE e.offload_at < NOW()
```

Comment:
now we want to poll all payloads in the WAL - we'll be able to remove the `offload_at` later